lib/new_relic/control/frameworks/grape.rb

```diff
@@ -0,0 +1,20 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'new_relic/control/frameworks/ruby'
+module NewRelic
+  class Control
+    module Frameworks
+      # Contains basic control logic for Grape
+      class Grape < NewRelic::Control::Frameworks::Ruby
+        protected
+
+        def install_shim
+          super
+          ::Grape::API::Instance.class_eval { include NewRelic::Agent::Instrumentation::ControllerInstrumentation::Shim }
+        end
```

Comment:
This is adding new functionality I don't think Grape users need. I would prefer to remove this method from the class. Allowing Grape framework applications to inherit from the Ruby framework class will keep the experience consistent with what it was before.

```suggestion
```